docs/templating/templates/view_matcher_reference.md

```diff
@@ -217,3 +222,39 @@ match:
 match:
     '@Ibexa\Contracts\ProductCatalog\ViewMatcher\LocationBased\RootLocation': true
 ```
+
+## Taxonomy entry ID
+
+`Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Id` matches based on an ID of the taxonomy entry.
+
+``` yaml
+match:
+    '@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Id': [1, 2, 3]'
+```
+
+## Taxonomy entry identifier
+
+`Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Identifier` matches based on an identifier of the taxonomy entry.
+
+``` yaml
+match:
+    '@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Identifier': ['spring', 'events', 'devices']
+```
+
+## Taxonomy entry level
```

Comment:
Here we should documented supported syntax for matcher value

Reply:
What do you mean, would you like to see both usage examples here? Like so?

```
taxonomy_entry_level:
  # ...
  match:
    '@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Level': 2
taxonomy_entry_level_gt_2:
  # ...
  match:
    '@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Level': '> 2'
```

Reply:
Is `'@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Level': [2, 3]` supported?
In the case of `'> 2'`-like syntax, what are the other supported operators? I guess I can write `'@Ibexa\Taxonomy\View\Matcher\TaxonomyEntryBased\Level': '<= 2'` but all possible operators should be listed to be sure.